### ISSUE TITLE
Define the term "redirect" in HTTP terms

### ISSUE THREAD

**Issue by tobie** [w3c/tr-links#5]
From the proposal, it seems you're talking about 301. After offline discussion, it seems you're implying 200 with some server side aliasing which, from a consumer's perspective is an implementation detail.

**Comment by tobie** [comment]
Note that from an SEO perspective (a key issue: people Google specs and end-up on old ones all the time), having a canonical spec which other URLs redirect to might be much more effective.

**Comment by deniak** [comment]
For /TR/shortname, it's currently an alias to the spec so it's indeed a HTTP 200. I wasn't planning to change that behavior.
However, for the new links (/TR/shortname/latest, /TR/shortname/stable and /TR/shortname/ed), we can definitely have a 301 or 307.

**Comment by deniak** [comment]
@dontcallmedom or @plehegar might also have an opinion in terms of SEO.

**Comment by tobie** [comment]
I'm concerned about the profileration of identical documents under different URLs. Maybe using https://tools.ietf.org/html/rfc6596 would help with this?

**Comment by dontcallmedom** [comment]
yes - @plehegar has asked me to look into initial SEO strategy for TR documents, and `rel=canonical` is the first item on my list. I had already submitted a [patch to respec](https://github.com/w3c/respec/pull/938) for that purpose a few months ago.

**Comment by tobie** [comment]
"canonical" can also be handled by HTTP headers, no?

**Comment by dontcallmedom** [comment]
yes; but it's likely to be much easier to manage it at the document level rather than at the HTTP headers level.

**Comment by tobie** [comment]
Have you filed something similar against bikeshed?

**Comment by dontcallmedom** [comment]
not yet - before doing that, I wanted to take a broader look at SEO-related changes we would like to make and bring it for discussion to spec-prod.
